src/style.scss

```diff
@@ -107,55 +112,24 @@ a,
   }
 }
 
+// TODO: Move this and ::placeholder to a new BaseInput component
+.input {
+  @apply outline-none text-skin-link bg-transparent border-none;
+}
+
 ::placeholder {
   color: var(--text-color);
   opacity: 0.6 !important;
 }
 
-.input {
-  outline: none;
-  color: var(--link-color);
-  background-color: transparent;
-  border: none;
-}
-
+// TODO: Move this to a new BaseTextarea component
 textarea {
   padding-top: 16px;
   font-size: 20px;
   line-height: 1.4em;
 }
 
-::-webkit-scrollbar-thumb {
-  background-color: var(--border-color);
-  background-clip: padding-box;
-  border: 0;
-  border-radius: 0;
-
-  &:hover {
-    background-color: var(--border-color);
-  }
-}
-
-.eyebrow {
-  text-transform: uppercase;
-  letter-spacing: 1px;
-  font-size: 16px;
-}
-
-select {
-  background-color: var(--bg-color) !important;
-}
-
-.fade-enter-active,
-.fade-leave-active {
-  transition: opacity 0.2s ease;
-}
-
-.fade-enter-from,
-.fade-leave-to {
-  opacity: 0;
-}
```

Comment:
These are needed for Vue's transition component.

https://github.com/snapshot-labs/snapshot/blob/95134e2ad7ff7e4d3a7cbf1f804fe6d490d4e434/src/components/Notifications.vue#L11
https://github.com/snapshot-labs/snapshot/blob/95134e2ad7ff7e4d3a7cbf1f804fe6d490d4e434/src/components/Ui/Modal.vue#L21

https://vuejs.org/guide/built-ins/transition.html#css-based-transitions

Reply:
Oh nice one